code/__DEFINES/reactions.dm

```diff
@@ -1,56 +1,248 @@
+#define FIRE_TRITIUM_ENERGY_RELEASED FIRE_HYDROGEN_ENERGY_RELEASED
+/// The minimum number of moles of trit that must be burnt for a tritium fire reaction to produce a radiation pulse. (0.01 moles trit or 10 moles oxygen to start producing rads.)
+#define TRITIUM_RADIATION_MINIMUM_MOLES 0.1
+/// A scaling factor for the range of radiation pulses produced by tritium fires.
+#define TRITIUM_RADIATION_RANGE_DIVISOR 1.5
+/// A scaling factor for the irradiation threshold of radiation pulses produced by tritium fires.
+#define TRITIUM_RADIATION_THRESHOLD_BASE 15
+
+// - Freon:
+/// The maximum temperature freon can combust at.
+#define FREON_MAXIMUM_BURN_TEMPERATURE 283
+/// The minimum temperature freon can combust at.
+#define FREON_LOWER_TEMPERATURE 60
+/// Multiplier for freonfire with O2 moles * FREON_OXYGEN_FULLBURN for the maximum fuel consumption
+#define FREON_OXYGEN_FULLBURN 10
+/// The maximum fraction of the freon in a mix that can combust each reaction tick.
 #define FREON_BURN_RATE_DELTA 4
-#define FIRE_FREON_ENERGY_RELEASED -300000 //amount of heat absorbed per mole of burnt freon in the tile
+/// The amount of heat absorbed per mole of freon burnt.
+#define FIRE_FREON_ENERGY_CONSUMED 3e5
+/// The maximum temperature at which freon combustion can form hot ice.
+#define HOT_ICE_FORMATION_MAXIMUM_TEMPERATURE 160
+/// The minimum temperature at which freon combustion can form hot ice.
+#define HOT_ICE_FORMATION_MINIMUM_TEMPERATURE 120
+/// The chance for hot ice to form when freon reacts on a turf.
+#define HOT_ICE_FORMATION_PROB 2
 
-#define N2O_DECOMPOSITION_MIN_ENERGY 1400
-#define N2O_DECOMPOSITION_ENERGY_RELEASED 200000
+// N2O:
+/// The minimum temperature N2O can form from nitrogen and oxygen in the presence of BZ at.
+#define N2O_FORMATION_MIN_TEMPERATURE 200
+/// The maximum temperature N2O can form from nitrogen and oxygen in the presence of BZ at.
+#define N2O_FORMATION_MAX_TEMPERATURE 250
+/// The amount of energy released when a mole of N2O forms from nitrogen and oxygen in the presence of BZ.
+#define N2O_FORMATION_ENERGY 10000
 
-#define NITRIUM_DECOMPOSITION_ENERGY 30000
+/// The minimum temperature N2O can decompose at.
+#define N2O_DECOMPOSITION_MIN_TEMPERATURE 1400
+/// The maximum temperature N2O can decompose at.
+#define N2O_DECOMPOSITION_MAX_TEMPERATURE 100000
+/// The maximum portion of the N2O that can decompose each reaction tick. (50%)
+#define N2O_DECOMPOSITION_RATE_DIVISOR 2
+/// One root of the parabola used to scale N2O decomposition rates.
+#define N2O_DECOMPOSITION_MIN_SCALE_TEMP 0
+/// The other root of the parabola used to scale N2O decomposition rates.
+#define N2O_DECOMPOSITION_MAX_SCALE_TEMP 100000
+/// The divisor used to normalize the N2O decomp scaling parabola.
+#define N2O_DECOMPOSITION_SCALE_DIVISOR ((-1/4) * ((N2O_DECOMPOSITION_MIN_SCALE_TEMP - N2O_DECOMPOSITION_MAX_SCALE_TEMP)**2))
```

Comment:
What?

Reply:
This is the value of the apex/nadir of the parabola created by the quadratic bit of the N2Os scaling factor. It's just being used to scale the whole thing so that it maxes out at 1 instead.